sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Release History
 
+## 1.0.0-beta.5 (Unreleased)
+### Added
+- Support for three more types of chat message content
+- New classes
+    - CommunicationError
+    - CommunicationErrorResponse
+    - CommunicationErrorResponseException
+
+## 1.0.0-beta.4 (Unreleased)
+### Added
+- Support for Rich Text Chat message content
+- New classes
+    - ChatMessageContent
+    - ChatMessageType 
+    - AddChatParticipantsErrors
+    - AddChatParticipantsResult
+    - ChatMessageType
+    
+### Breaking Changes
+- ChatMessage properties are now all required
+- ChatMessage type is no longer a String type but an extendable Enum type, ChatMessageType
+- ChatMessage content is no longer a String type but an object of ChatMessageContent
+- All OffsetDateTime properties are now in RFC3339 format instead of ISO8601 format
+
 ## 1.0.0-beta.3 (Unreleased)
```

Comment:
3 and 4 are not released, but we kept it so that we know the big iterations we have done on the sdk. If it is not a serious issue, I would like to keep it.

Reply:
We'd address that in a separate PR